packages/formik/src/Formik.tsx

```diff
@@ -51,6 +52,11 @@ type FormikMessage<Values> =
       payload: FormikState<Values>;
     };
 
+const defaultParseFn = (value: unknown, _name: string) => value;
+
+const defaultFormatFn = (value: unknown, _name: string) =>
```

Comment:
Supporting the decoupling of checkbox / radio functionality from handleChange, this could format `formValue: boolean => fieldProps.value` in the case of checkboxes / radios.